reuseport/reuseport.go

```diff
@@ -9,23 +9,12 @@
 package reuseport
```

Comment:
Can you change the top line of this file to `// +build !windows` as well please.

Reply:
Done